.github/workflows/CD.yml

```diff
@@ -0,0 +1,116 @@
+name: CD
+
+on:
+  push:
+    tags:        
+      - '**' 
+
+jobs:
+  validate-tag:
+    name: Validate tag
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout source
+        uses: actions/checkout@v4
+        with:
+          fetch-tags: true
+          fetch-depth: 0
+      
+      - name: Check no jumps in tags
+        run: |
+          # If a previous tag is X.Y, valid tags are only X.(Y+1) or (X+1).0
+          if [[ $(git tag -l | wc -l) != 1 ]]; then # If there is only one tag, no need to check
+            prev=($(git tag -l --sort=-v:refname | head -n 2 | tail -n 1 | tr "." " "))
```

Comment:
I think we might need an example output of this command, something like:

```
# this command splits the previous tag into a list of version components, eg for `12.2` it is the array (12 2).
```

Reply:
Fixed in [210e00add46f0c74d626e77b540e276a4d8b6609](https://github.com/ACCESS-NRI/replace_landsurface/pull/45/commits/210e00add46f0c74d626e77b540e276a4d8b6609)